onnx2tf/onnx2tf.py

```diff
@@ -454,6 +454,7 @@ def convert(
             print(f'{Color.REVERCE}Automatic generation of each OP name started{Color.RESET}', '=' * 40)
         op_name_auto_generate(
             input_onnx_file_path=f'{input_onnx_file_path}',
+            onnx_graph=onnx_graph,
```

Comment:
It works fine without adding this one line. 

https://github.com/onnx/models/raw/main/vision/body_analysis/emotion_ferplus/model/emotion-ferplus-8.onnx

```
onnxsim emotion-ferplus-8.onnx emotion-ferplus-8.onnx
```
![image](https://user-images.githubusercontent.com/33194443/207111130-515689a9-1de3-4cad-ab63-2c0e8d3dfc23.png)
```
onnx2tf -i emotion-ferplus-8.onnx
```
```
saved_model output started ==========================================================
saved_model output complete!
WARNING:absl:Please consider providing the trackable_obj argument in the from_concrete_functions. Providing without the trackable_obj argument is deprecated and it will use the deprecated conversion path.
Estimated count of arithmetic ops: 1.753 G  ops, equivalently 0.877 G  MACs
Float32 tflite output complete!
Estimated count of arithmetic ops: 1.753 G  ops, equivalently 0.877 G  MACs
Float16 tflite output complete!
```

Reply:
I do not understand the objection, as it should not work when calling the top level convert with an onnx_graph, and not with a file name. I call convert from a scrip like this:

```
onnx_model = onnx.load(onnx_file)
onnx2tf.convert(onnx_graph=onnx_model)
```
Per documentation, this is allowed

https://github.com/PINTO0309/onnx2tf/blob/8156a5a44a52702722e64943dc9f48b60a945575/onnx2tf/onnx2tf.py#L455-L459
then calls generate with input_onnx_file_path==''  (there is no file name).

https://github.com/PINTO0309/sng4onnx/blob/f24685387716ff64751da81d72b8a9496784fbde/sng4onnx/onnx_opname_generator.py#L35-L40
then has input_onnx_file_path=='' and onnx_graph==None (because of default argument)

https://github.com/PINTO0309/sng4onnx/blob/f24685387716ff64751da81d72b8a9496784fbde/sng4onnx/onnx_opname_generator.py#L69
is therefore True, and the script exits with an error.

My change passes both the file and the graph to generate, making this work

Reply:
Ah, I finally understand. It's a bug on the `sng4onnx` side. I will fix the `sng4onnx` side. This is the first time I have seen an example of a conversion using Python code. And it was my lack of testing.

Reply:
You were right. Thank you.